crs_msgs/srv/LocalizeToPart.srv

```diff
@@ -0,0 +1,5 @@
+sensor_msgs/PointCloud2[] point_clouds
+---
+bool success
+string error
+geometry_msgs/TransformStamped[] transforms
```

Comment:
Shouldn't the response be just a single transform?

Reply:
Yes, I will get this updated